lib/formtools/templates/index.js

```diff
@@ -1,12 +1,6 @@
-var fs = require('fs'),
-    handlebars = require('handlebars');
+var handlebars = require('handlebars'),
+    path = require('path');
 
-var documents = fs.readFileSync(__dirname + '/documents.html', 'utf-8'),
-    overview = {};
-
-overview.summary = fs.readFileSync(__dirname + '/overview/summary.html', 'utf-8');
+var documents = path.resolve(__dirname, '/documents.html');
```

Comment:
done. reverted to original code then replaced `__dirname + '/overview/summary.html'` with `path.resolve(__dirname, 'documents.html')`.